python/test_requirements.txt

```diff
@@ -1,6 +1,7 @@
-pytest~=3.1.2
-pytest-cov~=2.5.1
-cov-core~=1.15.0
-coverage~=4.4.1
+pytest
+pytest-cov
+pytest-flakes
+coverage
 tox
 pyyaml
+numpy
```

Comment:
What is the numpy req used for?

Reply:
In order for pytest to load `python/sbp/client/util/fftmonitor.py`, which currently has 0% test coverage